docs/native-tokens/minting-nfts.md

```diff
@@ -370,46 +373,45 @@ echo $slotnumber
 echo $script
 ```
 
-If everything is set, run this command to generate a raw transaction file.
+If everything is set, run the following command:
 
 ```bash
-cardano-cli transaction build-raw \
---fee $fee  \
---tx-in $txhash#$txix  \
+cardano-cli transaction build \
+--mainnet \
+--alonzo-era \
+--tx-in $txhash#$txix \
 --tx-out $address+$output+"$tokenamount $policyid.$tokenname" \
+--change-address $address \
 --mint="$tokenamount $policyid.$tokenname" \
 --minting-script-file $script \
 --metadata-json-file metadata.json  \
 --invalid-hereafter $slotnumber \
+--witness-override 2 \
 --out-file matx.raw
 ```
 
-As with every other transaction, we need to calculate the fee and the output and save them in the corresponding variables (currently set to zero).
+If the above command generate an output similar to:
```

Comment:
Reads incorrectly

Reply:
The suggestion was only for flow of the sentence (`If ... : ` felt like there is a continuation below, all good otherwise)

```suggestion
The above command may generate output as per below:
```